docs/token-incentives.md

```diff
@@ -0,0 +1,93 @@
+---
+id: token-incentives
+title: PSY Token
+sidebar_label: token-incentives
+slug: /psy-token
+---
+
+_**NOTE: NO PSY TOKEN HAS BEEN MINTED OR IS CIRCULATING AT THE MOMENT. 
+ANY TOKEN CURRENTLY CLAIMING TO BE PSY IS A SCAM**_
+
+_**THE INFORMATION LAID OUT HERE ARE TENTATIVE PLANS FOR A FUTURE TOKEN**_
+
+This document does its best to explain where PsyOptions needs community contributors 
+and how those contributions are rewarded.
+
+PsyOptions aims to foster an engaging and collaboritive community, join our 
+[discord](https://discord.com/invite/MgDdJKgZJc) or [telegram](https://t.me/psyoptions) 
+to get involved in the conversations! Note: The core contributors are most active in Discord.
+
+## Developer Contributions
+Contribute code, get PSY!
+
+### Grants
+If you have an idea for some sort of protocol, UI Component, or anything else that is useful
+for the PsyOptions ecosystem please reach out to developers@psyoptions.io. PsyOptions is actively 
+looking to partner and reward developers that want to build with us. 
+
+#### Examples
+
+##### Structured Products
+Any sort of structured product that leverages the PsyOptions protocol.
+
+##### AMM
+An AMM that uses contracts from PsyOptions.
+
+### Bounties
+Below is a list of bounties PsyOptions is offering for developers in our community to help out. 
+If you're interested in working on, have completed one, etc. please reachout to 
+developers@psyoptions.io so the list can update accordingly!
+
+| Bounty Title | Short Description | More Info | PSY Allocated |
+| ------------ | ----------------- | --------- | :-----------: |
+| Borrow on the Writer Token | A borrow lending protocol could allow the Writer Token to used as collateral, so option writers can get back some of their underlying. | Detailed ticket coming soon | TBD |
+| Multi-legged strategy UI component | A react component that leverages the PsyOptions bindings to display payoffs and execute multi-legged strategies | Detailed ticket coming soon | TBD | 
+
+## DeFi Thought Leaders
+The core contributors are incredible engineers, but are all retail traders. There are ideas 
+on where PsyOptions should go, but this is a community led project. PsyOptions leans on the 
+community to contribute feedback, ideas, and incredible visions about what the next iteration of 
+PsyOptions should be. 
+
+If you want to get involved and help improve PsyOptions, please join the 
+[discord](https://discord.com/invite/MgDdJKgZJc) and express your interest in becoming a 
```

Comment:
I'm weary of the "join discord and express your interest". I think this is fine for now and shouldn't block putting this out, but wondering if there's a better call to action. Otherwise we're just going to get a bunch of people saying they're interested, but not showing so with action